modules/opr/opr.go

```diff
@@ -7,7 +7,7 @@ const (
 	_ Type = iota
 	// V1 is JSON
 	V1
-	// V2 is Protobuf
+	// V2 + V3 is Protobuf
 	V2
```

Comment:
Maybe we should rename this to `Json` and `Proto` like they were originally. Or I should copy the Proto for a v3?

Reply:
the OPR type is different from the opr version, theoretically a json opr can hold all 3 versions and vice versa. i think it's fine to leave them as is. maybe change the comment to "V2 is for grading V2 & V3"